res/css/views/rooms/_MessageComposer.pcss

```diff
@@ -235,11 +236,12 @@ limitations under the License.
 
         &.mx_MessageComposer_closeButtonMenu {
             &::after {
-                background: rgba($accent, 0.1);
+                background: $accent-300;
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes